code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -245,6 +245,39 @@
 			L.extract_cooldown = max(0, L.extract_cooldown - 20)
 	..()
 
+
+/datum/reagent/teslium/energized_jelly/energized_ooze
+	name = "Energized Ooze"
+	description = "Electrically-charged Ooze. Boosts Oozeling's nervous system, but only shocks other lifeforms."
+	reagent_state = LIQUID
+	color = "#CAFF43"
+	taste_description = "slime"
+	overdose_threshold = 30
+
+/datum/reagent/teslium/energized_jelly/energized_ooze/on_mob_life(mob/living/carbon/M)
+	if(isjellyperson(M))
+		shock_timer = 0 //immune to shocks
+		M.AdjustAllImmobility(-40, FALSE)
+		M.adjustStaminaLoss(-2, 0)
+		if(isluminescent(M))
+			var/mob/living/carbon/human/H = M
+			var/datum/species/jelly/luminescent/L = H.dna.species
+			L.extract_cooldown = max(0, L.extract_cooldown - 20)
+	if(isoozeling(M))
+		shock_timer = 0 //immune to shocks
+		M.AdjustAllImmobility(-40, FALSE)
+		M.adjustStaminaLoss(-2, 0)
+	..()
+
+/datum/reagent/teslium/energized_jelly/overdose_process(mob/living/carbon/M)
+	shock_timer++
+	if(isoozeling(M) || isjellyperson(M))
+		shock_timer = (rand(0,2))
+		M.electrocute_act(rand(5,20), "Teslium in their body", 1, 1) //Override because it's caused from INSIDE of you
```

Comment:
```suggestion
		M.electrocute_act(rand(5,20), "Energized Jelly in their body", 1, 1) //Override because it's caused from INSIDE of you
```
Tiny oversight here

Reply:
Ay thanks, didnt even realise